odoo/tests/common.py

```diff
@@ -1812,6 +1812,17 @@ def route_profiler(request):
             return sup.profile(description=request.httprequest.full_path)
         return profiler.Nested(_profiler, patch('odoo.http.Request._get_profiler_context_manager', route_profiler))
 
+    def make_rpc(self, route, params, headers=None):
```

Comment:
move to prep PR, and replace existing code manually making json rpc to ensure the API fits the need

Reply:
and rename to contain json in the name